### DISCUSSION TITLE
Selected incorrect scenery directory and now program will not load.

### DISCUSSION THREAD

**Discussion by betterpilot** [kubilus1/autoortho#245] [Q&A] [answered]
### Check that box!

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: Windows 11
- XPlane Version: 11/12

### question

I selected the incorrect file directory for my custom scenery folder, and now every time I run AutoOrtho I get the attached error message with no way to reset the folder inside the app. 

Is there a way I can redirect the app to the custom scenery locations? I have not been able to get back to the GUI where the scenery location is set. 

![Screenshot_3](https://github.com/kubilus1/autoortho/assets/72628364/4cd88245-269f-4a3e-ba61-413af1c45914)

**Comment by kubilus1** [comment, marked as answer]
That message is just a warning.   For the gui not showing, maybe you
selected to hide it.   You can edit the config file directly '.autoortho'
under your users directory.

**Reply by betterpilot** [reply]
Thanks, editing the config restored the window which allowed me to point it to the right directory.

**Reply by Matteo230174** [reply]
Where is the file to edit? i have the same issue

**Reply by kubilus1** [reply]
The config file is '.autoortho' in your users dir.  So on Windows that would be 'C:\Users\<youruser\.autoortho'
